driver-core/src/main/java/com/datastax/driver/core/ExecutionInfo.java

```diff
@@ -147,4 +137,30 @@ public PagingState getPagingState() {
             return null;
         return new PagingState(this.pagingState, this.statement);
     }
+
+    /**
+     * Whether the cluster reached schema agreement within a given delay after
+     * the execution of this query.
+     *
+     * After a successful schema-altering query (ex: creating a table), the driver
+     * will check if the cluster's nodes agree on the new schema version. If not,
+     * it will keep retrying for a given delay (configurable via
+     * {@link Cluster.Builder#withMaxSchemaAgreementWaitSeconds(int)}).
+     * <p>
+     * If this method returns {@code false}, clients can call {@link Metadata#checkSchemaAgreement()}
+     * later to perform the check manually.
+     * <p>
+     * Note that the schema agreement check is only performed for schema-altering queries
+     * For other query types, this method will always return {@code true}.
```

Comment:
I think it's the most consistent behavior if you consider a generic client that executes queries without knowing if they're schema-altering or not, but wants to be notified if there was a schema agreement problem.